src/main/groovy/com/deploygate/gradle/plugins/tasks/UploadArtifactTask.groovy

```diff
@@ -116,12 +116,9 @@ abstract class UploadArtifactTask extends DefaultTask {
             if (!sent && (Config.shouldOpenAppDetailAfterUpload() || response.typedResponse.application.revision == 1)) {
                 BrowserUtils.openBrowser "${project.deploygate.endpoint}${response.typedResponse.application.path}"
             }
-        } catch (HttpResponseException e) {
+        } catch (Throwable e) {
```

Comment:
I understand to this change behavior to send to server always.
So, I ask to reason to why aggregate to super class.

Reply:
We never allow any failure when notifying lifecycle events to the server so it's better to suppress Throwable rather than specifying specific exceptions.

Reply:
OK, I understand to this change is necessary 👍